src/api/server.tsx

```diff
@@ -22,3 +24,11 @@ export async function addTodo(todo: any) {
   const actualData = response.json();
   return actualData;
 }
+
+export async function deleteTodo(id: any) {
+  const response = await fetch(BASE_URL + Url.DeleteTodo + id, {
+    method: "DELETE",
+  });
+  const responseOk = response.ok;
+  return responseOk;
```

Comment:
Couldn't you just return `response.ok`?

Actually you do that on all the async functions - setting to a temp and returning the temp - you can just return it directly.

Reply:
Sounds good.